src/main.cpp

```diff
@@ -1,8 +1,19 @@
 #include <QApplication>
+#include <QStringListModel>
+#include <QStandardItemModel>
+#include "projectviewerwindow.h"
 
 int main(int argc, char *argv[])
 {
     QApplication a(argc, argv);
 
+    QStandardItemModel *model = new QStandardItemModel;
```

Comment:
The model creates a memory leak here because nobody removes the object of it. There is 2 possible solution here: 
1. Give ownership of the model to project viewer window
2. Create a model on the stack, and it removes automatically. 

In future pay attention to this detail, as unnecessary objects will be stored in memory, when you do not need them.

Reply:
Ok.So I need to fix it and again send a request?